src/Worker/Handler/DebugTrait.php

```diff
@@ -0,0 +1,62 @@
+<?php
+declare (strict_types=1);
+
+namespace Maleficarum\Worker\Handler;
+
+/**
+ * Class DebugTrait
+ * @package Maleficarum\Worker\Handler
+ */
+trait DebugTrait {
+    /**
+     * Add debug message
+     *
+     * @param string $message
+     * @param array $options
+     *
+     * @return $this
```

Comment:
Incorrect notation - it returns an instance of a classes that uses the trait not a "$this".

Reply:
Changed at 5f3f302cb54de38384d4ba0180bb30fabe25ec7e